checks.d/couchbase.py

```diff
@@ -172,7 +172,12 @@ def get_data(self, server, instance):
         url = '%s%s' % (server, COUCHBASE_STATS_PATH)
 
         # Fetch initial stats and capture a service check based on response.
-        service_check_tags = ['instance:%s' % server]
+        service_check_tags = instance.get('tags', [])
+        if service_check_tags is None:
+            service_check_tags = []
+        else:
+            service_check_tags = list(set(service_check_tags))
```

Comment:
i think you can replace all that with `service_check_tags = list(set(instance.get('tags') or []))`